.github/workflows/wordpress-org-asset-update.yml

```diff
@@ -0,0 +1,25 @@
+name: Plugin asset/readme update
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'readme.md'
+      - '.wordpress-org/**'
+
+jobs:
+  update:
+    name: Update
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: WordPress.org plugin asset/readme update
+        uses: wearerequired/action-wordpress-plugin-asset-update@copy-only
+        env:
+          README_NAME: 'README.md'
```

Comment:
Needs to be `'readme.md'`, see https://github.com/wearerequired/digest/runs/2691251020#step:4:73